Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "ChainRulesCore"
 uuid = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
-version = "0.6.1"
+version = "0.6.1-DEV"
```

Comment:
Should be changed to `0.7.0` or `0.7.0-DEV`

Do we want to release this change as `0.7.0`? Or do we want to keep it as `0.7.0-DEV` until e.g. we can add the macro that makes frules nice?

Reply:
Adding the macro isn't a breaking change, so I don't see why we can't just release it tbh. Would help move things forwards

Reply:
```suggestion
version = "0.7.0"
```